src/helpers/index.js

```diff
@@ -46,6 +47,24 @@ module.exports.buildPrettifier = prettierConfig => text => (
   prettier.format(text, prettierConfig)
 );
 
+const componentTypes = {
+  FUNCTIONAL: 'functional',
+  CLASS: 'class',
+  PURE_CLASS: 'pure-class',
+};
+module.exports.componentTypes = componentTypes;
+
+const stylingTypes = {
+  STYLED: 'styled-components',
+  CSS_MODULES: 'css-modules',
+  APHRODITE: 'aphrodite',
+};
```

Comment:
I think we should consider `none` a valid styling type.

My reasoning is that the default styling is no styling. Someone may install this project, skim the README too quickly, and not even realize that it's capable of doing styling!

By showing that "None" is the styling option they've selected, we also show the options they *haven't* selected. They might say "oh cool this does styled-components! I need that".

Reply:
OK! When you said you want to keep the project as simple as possible, I thought showing the `styling: none  styled  css-modules aphrodite` in console every time would confuse beginner users.

Reply:
Yeah, my fault for not specifying! I can see the argument of wanting to reduce the noise as much as possible, but I think that ultimately it's worth the added line of info :)